### ISSUE TITLE
Tab logical order is wrong between the object menu and main content page

### ISSUE THREAD

**Issue by mathisscott** [vmware/clarity#1947]
URL: http://clarity-ks.surge.sh/alerts

Reproduced Steps:
1. Go to Alerts page
2. Use tab button browse the page.

Actual Results:
The tab logical order is wrong. It point to main content page, then point to the object menu at left of the page.

Expected Results:
The tab order should keep from left to right, and here it should point to the object menu before point to the main content page.

Violation of Section 508 Standards: 
1194.31(a) At least one mode of operation and information retrieval that does not require user vision shall be provided, or support for Assistive Technology used by people who are blind or visually impaired shall be provided. 
1194.31(b) At least one mode of operation and information retrieval that does not require visual acuity greater than 20/70 shall be provided in audio and enlarged print output working together or independently, or support for Assistive Technology used by people who are visually impaired shall be provided. 
User Impact: Blind and disable with hand.

NOTE: need to know if this is an implementation issue or a Clarity issue.

**Comment by mathisscott** [comment]
duplicate of #1923
